sintia/sintia.py

```diff
@@ -503,6 +503,8 @@ def format_quote(self, quote: Quote) -> str:
             'k': 'kelvin',
             'mi': 'miles',
             'km': 'kilometers',
+            'lbs': 'pounds',
+            'kg': 'kilograms',
```

Comment:
wanna add just kilos as well? You don't have to

Reply:
no, i think everyone can understand kg or kilograms instead of kilos

Reply:
![Why not both?](https://i.kym-cdn.com/entries/icons/facebook/000/006/759/both.jpg)

Reply:
:|